subsys/net/lib/dns/resolve.c

```diff
@@ -426,7 +426,7 @@ static inline void invoke_query_callback(int status,
  */
 static void release_query(struct dns_pending_query *pending_query)
 {
-	int busy = k_work_cancel_delayable(&pending_query->timer);
+	unsigned int busy = k_work_cancel_delayable(&pending_query->timer);
```

Comment:
Not sure if this change should actually be made - unless the signature of `k_work_cancel_delayable()` has also been changed.

Reply:
Yes, it has been changed.